cts/cts-cli.in

```diff
@@ -3280,7 +3281,7 @@ def results(regs, save, verbose):
             dest = "%s/cli/regression.%s.exp" % (test_home, r.name)
             copyfile(r.results_file, dest)
 
-        r.diff()
+        r.diff(verbose)
```

Comment:
Fun that we had this as an argument to `diff()` already and were never using it.

Side note, I was curious if Python has a diff utility so that we don't have to make a subprocess call at all in our `diff()` function. It does:
https://docs.python.org/3.6/library/difflib.html
(link is for 3.6 since that's our minimum supported version)

Reply:
That's good to know.  This is actually not the only place we are diffing files.  There's another one in cts-scheduler (and there'll be another one should we ever convert cts-schemas to python).  I'm intending to get around to combining this code one day (see https://projects.clusterlabs.org/T970).

Reply:
> `cts-schemas` to python

That's certainly the plan: [T156](https://projects.clusterlabs.org/T156)

> combining this code
Please:
* Make the task editable by Pacemaker Developers
* Consider adding a note about difflib

Reply:
I don't know why everything defaults to private.  I guess I can change that.